platform/iotex/model.go

```diff
@@ -0,0 +1,39 @@
+package iotex
+
+const TransferFee = "10000000000000000"
```

Comment:
is it possible to get transfer fee from the transaction object and not to hardcode?

Reply:
our transfer fee is fixed
for contract execution it will varying depending on specific inputs

Reply:
can we do `gasLimit * gasPrice` in this case? we should avoid hardcoding any values

Reply:
fixed in latest push